src/Bundle/Maker/MakeGrid.php

```diff
@@ -0,0 +1,167 @@
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:grid';
+    }
+
+    public static function getCommandDescription(): string
+    {
+        return 'Creates a Grid for a Doctrine entity class';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig): void
+    {
+        $command
+            ->setDescription(self::getCommandDescription())
+            ->addArgument('entity', InputArgument::OPTIONAL, 'Entity class to create a grid for')
+            ->addOption('namespace', null, InputOption::VALUE_REQUIRED, 'Customize the namespace for generated factories', 'Grid')
+        ;
+
+        $inputConfig->setArgumentAsNonInteractive('entity');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void
+    {
+        if ($input->getArgument('entity')) {
+            return;
+        }
+
+        $argument = $command->getDefinition()->getArgument('entity');
+        $entity = $io->choice($argument->getDescription(), $this->entityChoices());
+
+        $input->setArgument('entity', $entity);
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
+    {
+        $class = $input->getArgument('entity');
+
+        if (!\class_exists($class)) {
+            $class = $generator->createClassNameDetails($class, 'Entity\\')->getFullName();
+        }
+
+        if (!\class_exists($class)) {
+            throw new RuntimeCommandException(\sprintf('Entity "%s" not found.', $input->getArgument('entity')));
+        }
+
+        $namespace = $input->getOption('namespace');
+
+        // strip maker's root namespace if set
+        if (0 === \mb_strpos($namespace, $generator->getRootNamespace())) {
+            $namespace = \mb_substr($namespace, \mb_strlen($generator->getRootNamespace()));
+        }
+
+        $namespace = \trim($namespace, '\\');
+
+        $entity = new \ReflectionClass($class);
+        $grid = $generator->createClassNameDetails($entity->getShortName(), $namespace, 'Grid');
+
+        $repository = new \ReflectionClass($this->managerRegistry->getRepository($entity->getName()));
+
+        if (0 !== \mb_strpos($repository->getName(), $generator->getRootNamespace())) {
+            // not using a custom repository
+            $repository = null;
+        }
+
+        $this->defaultFieldsFor($entity->getName());
+
+        $generator->generateClass(
+            $grid->getFullName(),
+            __DIR__.'/../Resources/config/skeleton/Grid.tpl.php',
+            [
+                'entity' => $entity,
+                'defaultFields' => $this->defaultFieldsFor($entity->getName()),
+                'repository' => $repository,
+            ]
+        );
+
+        $generator->writeChanges();
+
+        $this->writeSuccessMessage($io);
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies): void
+    {
+        // noop
```

Comment:
?

Reply:
It's required by the interface `Symfony\Bundle\MakerBundle\MakerInterface'.
But I have to check the return typehint on Symfony 6.

Reply:
No that's good, that's not part of Symfony monorepository
https://github.com/symfony/maker-bundle/blob/main/src/MakerInterface.php

Reply:
https://github.com/symfony/maker-bundle/blob/006577909ded549ee48fd69aa4da1a75ff9a2b55/src/Maker/MakeSerializerNormalizer.php

Reply:
I meant - what do you mean by noop? No operation? I have the feeling, that both of us should be more specific :) I with my question and you with the comment here, so people would not need to try to resolve what is `noop`

Reply:
I don't know lol
That's just copied/paste from https://github.com/zenstruck/foundry/blob/master/src/Bundle/Maker/MakeFactory.php#L170